### DISCUSSION TITLE
Flickering under low brightness

### DISCUSSION THREAD

**Discussion by marcosvrs** [awawa-dev/HyperHDR#69] [Q&A] [answered]
Using NodeMCU v3 with HyperSerialWLED connected to my NUC with USB cable, colors under low brightness generate a flickering on my SK9822 LEDs.
I'm using an 5V 20A power supply connected on both sides of my 180 LED strip.
Using the normal WLED Effects, no flickering is visible.
Using the HyperHDR effects I can see the flickering.
Already tried multiple configs like Smoothing, Blackbar detection and Colored Backlights.

Any ideas?

**Comment by awawa-dev** [comment, marked as answer]
First disable all the unnecessary things just like Smoothing and set minimal backlight to 0 (disable) at first place.
If you put a STATIC low brightness color using HyperHDR's remote the flickering still manifests? Was WLED disconnected from NUC when you played effects without flickering? Just to be sure: power limit in WLED is disabled?

**Reply by marcosvrs** [reply]
Disabling the WLED power limit was the solution! Thank you very much!

**Comment by marcosvrs** [comment]
As feedback, I missed a public docker image, so I can update my current image automatically when you push a new version (Watchtower).

**Comment by erganmedia** [comment]
Why is it important to disable power limit? I want to use a power supply which only provides about 10A. If I don't use the power limiter of WLED the LEDs could try to use more than 10A. I don't want to deal with possible hardware issues, so I have enabled the power limiter. Is there a way to keep the limit and also prevent the flickering? With Hyperion.ng I had no issues btw, but with HyperHDR I do. I would like to keep HyperHDR thogh.

**Reply by awawa-dev** [reply]
Default WLED power limit is much lower than 10A. And default power limit definitely causing such issues. We provide RGB values for WLED, what's happening there is outside the scope of this project.
